docs/release-procedure.md

```diff
@@ -23,29 +24,25 @@ as reviewing them again would not add a lot of value but would add lots of exces
 Hence non-cherry-pick commits should happen in a separate branch in your own fork, and
 be delivered to the release branch by using a PR.
 
-Note that Modin uses fully signed commits at least for releases, so you have to have GPG keys set up. See [onboarding instructions](https://github.com/modin-project/modin/blob/master/onboarding/onboarding.md) on where to get started.
+Note that Modin uses fully signed commits, so you have to have GPG keys set up. See [onboarding instructions](https://github.com/modin-project/modin/blob/master/onboarding/onboarding.md) on where to get started.
 
 To update Modin version, follow the instructions below.
 
-### Update the link to release in readme
-
-**Note**: from now on you work in a branch in your fork (in `origin`).
-
-Change the `README.md` in the repo root so release badge points to new (would be created) release.
+### Prepare the repo for release
 
-### Commit
+**Note**: this should be done in your fork of Modin.
 
-        git commit -a -S -m "Release version X.Y.Z"
-
-Pay attention to that `-S` switch if you haven't enabled signing all the commits!
-Modin requires release commits to be signed (do not mix this with "signing off" commits, this
-is a whole different story - signed commits are cryptographically signed).
-
-Push the commit, make a PR _against `release-X.Y.Z`_ branch and, when it's merged, pull that branch from `upstream` before proceeding.
+First, update your fork of Modin's master with the main repo's master. From your master, create a new
+branch called `release-X.Y.0` off of master. Create an empty commit in your new branch with the message
+`Release version X.Y.0`. Make sure to sign this commit with both your GPG key
+and with the conventional `git commit -s` (so `git commit -s -S`). Open a PR against modin-project/modin with just this commit.
 
 ### Tag commit
 
-**Note**: from now on you work in the release branch (in `upstream`).
+After the PR has been merged, clone a clean copy of the Modin repo from the modin-project organization.
```

Comment:
This is not necessarily to have a clean copy. It is just enough to pull latest changes from upstream.

Reply:
This is true - but @devin-petersohn's original instructions ask for users to do it in a clean copy of the repo, and I think that makes sense to prevent any accidents!